### PR TITLE
Update README.md for Apple Privacy Manifest

### PR DESCRIPTION
**Pull request by buenaflor**
#skip-changelog

```[tasklist]
### Blocked by
- [ ] https://github.com/getsentry/sentry-docs/pull/9872
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ For detailed information on how to build and run the samples, check out our `REA
 [sentry-samples](https://github.com/getsentry/sentry-kotlin-multiplatform/tree/main/sentry-samples)
 folder.
 
+## Apple Privacy Manifest
+
+Starting with [May 1st 2024](https://developer.apple.com/news/?id=3d8a9yyh), iOS apps are required to declare approved reasons to access certain APIs. This also includes third-party SDKs.
+If you are using the Sentry Kotlin Multiplatform SDK with Apple device targets then update your Sentry Cocoa SDK to `8.21.0` minimum. However we recommend using `8.25.0` to ensure compatibility with Apple's guidelines.
```
Comment (kahest) [inline README.md:49]:
```suggestion
If you are using the Sentry Kotlin Multiplatform SDK with Apple device targets then update your Sentry Cocoa SDK to `8.25.0` minimum.
```

Comment (kahest) [reply README.md:49]: let's make it as easy as possible for our users :) if they can update to 8.21.0, they should have no problem updating to 8.25.0 as well. we'll also update the docs

Comment (kahest) [reply README.md:49]: another thought - let's add [docs](https://docs.sentry.io/platforms/flutter/data-management/apple-privacy-manifest/)+[troubleshooting](https://docs.sentry.io/platforms/flutter/troubleshooting/#missing-api-declaration-after-app-store-review) pages for KMP and link to them here?

Comment (buenaflor) [reply README.md]: https://github.com/getsentry/sentry-docs/pull/9872

File: README.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ For detailed information on how to build and run the samples, check out our `REA
 [sentry-samples](https://github.com/getsentry/sentry-kotlin-multiplatform/tree/main/sentry-samples)
 folder.
 
+## Apple Privacy Manifest
+
+Starting with [May 1st 2024](https://developer.apple.com/news/?id=3d8a9yyh), iOS apps are required to declare approved reasons to access certain APIs. This also includes third-party SDKs.
+If you are using the Sentry Kotlin Multiplatform SDK with Apple device targets then update your Sentry Cocoa SDK to `>=8.25.0`.
+For more information, please refer to our [Apple Privacy Manifest Guide](https://docs.sentry.io/platforms/kotlin-multiplatform/data-management/apple-privacy-manifest/).
```
Comment (kahest) [inline README.md:50]:
```suggestion
Starting with [May 1st 2024](https://developer.apple.com/news/?id=3d8a9yyh), apps submitted to the Apple App Store are required to declare approved reasons to access certain privacy-relevant APIs. This also includes usages of these APIs via third-party SDKs. To ensure compliance, update your Sentry Cocoa SDK to `>=8.21.0`.
For more information, refer to our [Apple Privacy Manifest Guide](https://docs.sentry.io/platforms/kotlin-multiplatform/data-management/apple-privacy-manifest/).
```

Comment (kahest) [reply README.md:50]: reasons for my suggestions: it's not just iOS apps, but should also apply to macOS, tvOS, etc. also it's not so much about targeting Apple devices, but submitting to the Apple App Store

**Review by kahest — APPROVED** [review]

**Pull request merged**